bor-mainnet.toml

```diff
@@ -453,3 +453,38 @@
 'v1-045000-045500-borspans.seg' = 'ae40aa774a8ce225d4b3b7a1b77592fc4373048c'
 'v1-045000-045500-headers.seg' = '5360bd1f51c275cb9844a16d70813a1440400d72'
 'v1-045000-045500-transactions.seg' = '669ebe60283b63e663e24e0aa7bc3e29c1d3d756'
+'v1-045500-046000-bodies.seg' = 'cca9947dbff5e228e92a748c27a3b7786548b71c'
+'v1-045500-046000-borevents.seg' = 'c5019b86f7551aa3c3fb98eaecb37c28016bc59f'
+'v1-045500-046000-borspans.seg' = 'c6135b143f99b5ee7e8941d05a42497e6e32420f'
+'v1-045500-046000-headers.seg' = '3e7f558763cec757db16df4caadb65e831b70508'
+'v1-045500-046000-transactions.seg' = 'dc7a59fc1737796d77197159ff7133e5e29413f9'
+'v1-046000-046500-bodies.seg' = '19e795b0df22454f043453eac14576542b8e7c9e'
```

Comment:
`borevents` and `borspans` are missing for 46m+

Reply:
right... ok... I think here is the reason: https://github.com/ledgerwatch/erigon/pull/9077